### PR TITLE
Add support for WebAssembly AppManifest.displayName

### PR DESCRIPTION
**Pull request by jeromelaban**

### CODE REVIEW

File: doc/articles/using-uno-ui.md [hunk 1]
```diff
@@ -75,6 +75,27 @@ the **Show all Files** icon at the top.
 
 If you notice an issue, or an error in the commented code of the generated file, you may need to alter you Xaml.
 
+## Configure the Manifest for the WebAssembly Head
+In you WASM head, create a folder named `WasmScripts`, with a file named
+`AppManifest.js` with the `Embedded resource` build action.
```
Comment (carldebilly) [inline doc/articles/using-uno-ui.md]: This filename is required ?  I thought it's just a recommendation.

File: doc/articles/using-uno-ui.md [hunk 2]
```diff
@@ -75,6 +75,27 @@ the **Show all Files** icon at the top.
...
+```
+
+The properties are :
+* **splashScreenImage**: defines the image that will be centered on the window during the application's loading time
+* **splashScreenColor**: defines the background color of the splash screen
+* **displayName**: defines the default name of the application in the browser's window title
+
 ## Supporting multiple Platforms in Xaml files
 
 The Uno.UI Xaml parser has the ability to manage specific namespaces, giving the ability to ignore or enable specific Xaml nodes and attributes.
```
Comment (carldebilly) [inline doc/articles/using-uno-ui.md]: XAML

File: doc/articles/using-uno-ui.md [hunk 3]
```diff
@@ -75,6 +75,27 @@ the **Show all Files** icon at the top.
...
+`AppManifest.js` with the `Embedded resource` build action.
+
+The manifest file can contain the following:
+
+```javascript
+var UnoAppManifest = {
+
+    splashScreenImage: "Assets/AppSplashScreen.scale-200.png",
+    splashScreenColor: "#3750D1",
+    displayName: "My Sample App"
```
Comment (carldebilly) [inline doc/articles/using-uno-ui.md]: And if we want a localized version?

File: src/Uno.UWP/UI/ViewManagement/ApplicationView.wasm.cs [hunk 4]
```diff
@@ -8,13 +8,13 @@ public string Title
...
 			{
-				const string command = "(function(){return document.title || \"\";})();";
+				const string command = "Uno.UI.WindowManager.current.getWindowTitle()";
 				return WebAssemblyRuntime.InvokeJS(command);
 			}
 			set
 			{
 				var escapedValue = WebAssemblyRuntime.EscapeJs(value);
-				var command = "(function(t){document.title = t; return \"\";})(\"" + escapedValue + "\");";
+				var command = "Uno.UI.WindowManager.current.getWindowTitle(\"" + escapedValue + "\");";
```
Comment (carldebilly) [inline src/Uno.UWP/UI/ViewManagement/ApplicationView.wasm.cs]: It's really `getWindowTitle()` ?? I suspect there's a cut&paste typo here.

File: doc/articles/using-uno-ui.md (hunk 3 above)
Comment (jeromelaban) [reply doc/articles/using-uno-ui.md]: No localization for now, and I'm thinking this file has high overlaps the PWA manifest file.

**Pull request merged**